elastic/integrations
# [Cloud Security ]add observer.vendor to cloud_security_posture package version 1.10

## Proposed commit message

Backport of the https://github.com/elastic/integrations/pull/10945 into currently released 1.10 package version as per requirement in https://github.com/elastic/kibana/issues/188802



## Checklist


## Author's Checklist



## How to test this PR locally



- build with `elastic-package build`
- run stack with this package via `elastic-package stack up -v -d`
- ingest data into findings and vulnerability datastream, check that `observer.vendor` is provided

## Related issues


- contributes to https://github.com/elastic/kibana/issues/188802

## Screenshots
